:q--- /usr/lib64/python3.6/site-packages/pyzotero/zotero.py	2019-06-12 08:24:13.756358950 -0400
+++ /usr/lib64/python3.6/site-packages/pyzotero/zotero.py	2019-06-12 08:26:29.932689020 -0400
@@ -1310,6 +1310,12 @@
         for idx, tmplt in enumerate(to_add):
             tmplt["title"] = os.path.basename(files[idx])
             tmplt["filename"] = files[idx]
+            if files[idx].endswith('.patch.txt'):
+                tmplt["contentType"] = 'text/plain'
+                tmplt["charset"] = 'utf-8'
+            elif files[idx].endswith('.xml'):
+                tmplt["contentType"] = 'text/xml'
+                tmplt["charset"] = 'utf-8'
         if parentid:
             return self._attachment(to_add, parentid)
         else:
